src/Homework3/8_Caesar_Cipher_(encryptor).py

```diff
@@ -0,0 +1,26 @@
+"""Caesar Cipher (encryptor).
+
+This mission is the part of the set. Another one - Caesar cipher decriptor.
+
+Your mission is to encrypt a secret message (text only, without special chars
+like "!", "&", "?" etc.) using Caesar cipher where each letter of input text
+is replaced by another that stands at a fixed distance.
+For example ("a b c", 3) == "d e f"
+
+"""
+
+
+def to_encrypt(text, delta):
+    alphabet = 'abcdefghijklmnopqrstuvwxyz'
+    encrypt_lst = [' ' if char == ' ' else alphabet[
+        (alphabet.index(char) + delta) % len(alphabet)] for char in text]

```

Comment:
If i solve this task I think i'll move alphabet to dict (key: shifted_key) and iterated text replacing key with shifted key. 
It seems for me to be more effective but i'm not exactly sure